### DISCUSSION TITLE
Comparison with BirdPi-Net?

### DISCUSSION THREAD

**Discussion by SenorFusion** [tphakala/birdnet-go#327] [Q&A] [answered]
Hi all,

I am not really a birder, but I am building a system for a Christmas present.

What I haven't seen mentioned anywhere are the primary differences between this project and Birdnet-pi. I'm trying to decide which would be best for my use case.

This will be installed at a relative's house with no ability for me to remote access or trouble shoot and not a super tech savvy user. For that reason I really love that birdnet-go has a docker release available.

If it were just me I would do both and play with it, but I don't have a lot of time before Christmas.

Which is the most fire-and-forget with least up-keep and reliability?

Is there a feature comparison somewhere?

are they even comparable projects?

Thanks!

**Comment by SenorFusion** [comment, marked as answer]
Of course, I just found that you[ answered this here](https://github.com/tphakala/birdnet-go/discussions/321).

**Comment by isZumpo** [comment]
It is really the perfect gift. Your relatives will be very happy to receive it! I installed it at my grandparents' place around May, and it has been running very hands off since then.
In terms of upkeep, I've only had issues when they have managed to unplug the mic when cleaning around the house. Which was easily fixed through a phone call. I do however have a VPN tunnel to their raspberry such that I can sometimes take a look at the health of it and perform updates etc.

**Reply by SenorFusion** [reply]
> It is really the perfect gift. Your relatives will be very happy to receive it! I installed it at my grandparents' place around May, and it has been running very hands off since then. In terms of upkeep, I've only had issues when they have managed to unplug the mic when cleaning around the house. Which was easily fixed through a phone call. I do however have a VPN tunnel to their raspberry such that I can sometimes take a look at the health of it and perform updates etc.

Thank you for the info!

 Did you ever use Birdnet-Pi? If so, what are your thoughts on the comparison as far as remote management and easy of use for less tech savvy folks?

**Reply by tphakala** [reply]
One of big issues with BIrdNET-Pi is how it records audio, it wears sd cards out really quickly and it is not uncommon for sd card to fail after few months of use. At minimum I would use high endurance sd card, but it just prolongs inevitable sd card failure.

**Reply by SenorFusion** [reply]
> One of big issues with BIrdNET-Pi is how it records audio, it wears sd cards out really quickly and it is not uncommon for sd card to fail after few months of use. At minimum I would use high endurance sd card, but it just prolongs inevitable sd card failure.

Understood! Thanks for the great project!

I _think_ the current fork of birdnet-pi that is being updated has moved away from this, or at least offered record to ram as an option:

https://github.com/Nachtzuster/BirdNET-Pi/commit/a4122b5bccca22b55cbdb76e201fdca1ddaf3260

**Reply by isZumpo** [reply]
> > It is really the perfect gift. Your relatives will be very happy to receive it! I installed it at my grandparents' place around May, and it has been running very hands off since then. In terms of upkeep, I've only had issues when they have managed to unplug the mic when cleaning around the house. Which was easily fixed through a phone call. I do however have a VPN tunnel to their raspberry such that I can sometimes take a look at the health of it and perform updates etc.
> 
> Thank you for the info!
> 
>  Did you ever use Birdnet-Pi? If so, what are your thoughts on the comparison as far as remote management and easy of use for less tech savvy folks?
> 

@tphakala covers the comparisons quite well in his answers in this and the other discussion thread.

I only tried birdnet-pi at the start of this year for a short while. At that point birdnet-pi was quite abandoned and none of the newer forks existed. The only thing I felt that birdnet-go was lackning in comparison to birdnet-pi at that point was images of the detected birds. Which I later added in a PR to birdnet-go. 

I did find the installation process of birdnet-pi annoying since it required executing a script that installed a bunch stuff directly on the os, I much prefer running software in containers. Also feels like birdnet-go has a more modern UI design that might be more natural for users.

Though, all of the above are more of my personal opinions, so if you have the chance, I would recommend to try out both and see which one you prefer.

**Comment by jkrauska** [comment]
As a software developer, I really appreciate the work put in to BirdNet-Go.  BirdNet-Pi is using much older technology (PHP scripting) and the UI mirrors that.  I started some work trying to clean up BirdNet-Pi, but after finding and testing Birdnet-Go, I am hoping to work on this instead.

Other technical things:
Works in docker (selfcontained/modular/portable) vs making a mess of your home directory on a pi.

I'm not finding realtime stream/spectograms yet (which I use for checking my mic settings), but so far I'm loving what I'm seeing.
